base_station/gui/src/components/AutonTask.vue

```diff
@@ -230,8 +229,6 @@ export default {
           this.RadioSignalStrength.signal_strength = msg.message.signal_strength.toFixed(1)
          }else if (msg.topic === '/autonomous') {
           this.Joystick = msg.message
-        } else if (msg.topic === '/rr_drop_complete') {
```

Comment:
Was there a reason for also including some removal of RR code in this PR? I thought you had removed all of it in an earlier one?